### ISSUE TITLE
Allow to define comment style

### ISSUE THREAD

**Issue by lukasoppermann** [amzn/style-dictionary#967]
There is a `commentStyle` proper in the formatter, however it seems that at least in css this can not be overwritten: https://github.com/amzn/style-dictionary/blob/main/lib/common/formatHelpers/createPropertyFormatter.js#L52

Would be great if this option could be added: https://amzn.github.io/style-dictionary/#/formats?id=cssvariables

`options.commentStyle` could either be `short` or `none` ot it could be a boolean.

**Comment by dbanksdesign** [comment]
I think that makes sense. It sounds like you are looking to turn off comments for specific files, maybe we expose that as an option on file configs?

**Comment by lukasoppermann** [comment]
Exactly, I want to turn it off. Exposing it like this would be perfect.

**Comment by jorenbroekema** [comment]
https://github.com/amzn/style-dictionary/commit/b9270b20f19749f3e8670f523580b132c76e7b29#diff-a8a4f9ccb2376c2ec73098072d8621be67c94e04cf9cc2e1b575a28aee5202e4R333 I added a feature for this, both in v3 and v4

**Comment by jorenbroekema** [comment]
Actually, it doesnt support "none", if you do that you might get `#ffffff; undefined` or something along those lines, should be easy to add though, PR welcome

**Comment by DarioSoller** [comment]
I have similar topic for the `fileHeaderCommentStyles`. But I better open a separate issue for that?

**Comment by DarioSoller** [comment]
Just for reference: https://github.com/style-dictionary/style-dictionary/issues/1312
